### PR TITLE
Prepare release 7.3.0

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 Please refer to [UPGRADING](UPGRADING.md) guide for upgrading to a major version.
 
+## 7.3.0 - 2021-03-21
+
+### Added
+
+- Support for DER and P12 certificates [#2413](https://github.com/guzzle/guzzle/pull/2413(
+- Support the cURL (http://) scheme for StreamHandler proxies [#2850](https://github.com/guzzle/guzzle/pull/2850)
+- Support for `guzzlehttp/psr7` [#2874](https://github.com/guzzle/guzzle/pull/2874)
```
Comment (GrahamCampbell) [inline CHANGELOG.md:11]: This should be omitted. I assume we don't want to allow 2.x in here until 2.0.0 stable is out. That could coincide with Guzzle 7.4.0.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Nyholm) [reply CHANGELOG.md:11]: Nope. 

That is exactly what we do want. We want to allow people to install it and report bugs before 2.0 is released.

Comment (GrahamCampbell) [reply CHANGELOG.md:11]: Then we need to update Guzzle's composer.json to allow `^2.0@dev`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 Please refer to [UPGRADING](UPGRADING.md) guide for upgrading to a major version.
 
+## 7.3.0 - 2021-03-21
+
+### Added
+
+- Support for DER and P12 certificates [#2413](https://github.com/guzzle/guzzle/pull/2413(
+- Support the cURL (http://) scheme for StreamHandler proxies [#2850](https://github.com/guzzle/guzzle/pull/2850)
+- Support for `guzzlehttp/psr7:2.0` [#2878](https://github.com/guzzle/guzzle/pull/2878)
```
Comment (GrahamCampbell) [inline CHANGELOG.md:11]: Maybe call it `^2.0`. Don't want people trying to copy this as a version constraint.

**Review by GrahamCampbell — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Nyholm) [reply CHANGELOG.md:11]:
```suggestion
- Support for `guzzlehttp/psr7:^2.0` [#2878](https://github.com/guzzle/guzzle/pull/2878)
```

Sure!

**Pull request merged**